specification/2.0/schema/buffer.schema.json

```diff
@@ -9,6 +9,7 @@
             "type": "string",
             "description": "The uri of the buffer.",
             "format": "uriref",
+            "minLength": 1,
```

Comment:
I think that such restriction was always implicit:
- `uri` is an _external_ reference;
- empty string URI points to the glTF file itself.

Following this logic, a valid glTF asset can't have empty string URIs.

If making such schema adjustment is impossible, we should document in the readme how to treat empty URIs.